app/models/defence_request.rb

```diff
@@ -119,8 +115,4 @@ def closed?
   def notify_interview_start_change
```

Comment:
I would suggest to remove this as well as you're on it.. Cause solicitors won't have access until much later. What do you think @ansonK ?